presto-docs/src/main/sphinx/admin/function-namespace-managers.rst

```diff
@@ -0,0 +1,34 @@
+===========================
+Function Namespace Managers
+===========================
+
+.. warning::
+
+    This is an experimental feature being actively developed. The way
+    Function Namespace Managers are configured might be changed.
+
+Function namespace managers support storing and retrieving user-defined
+functions with a persistent storage, allowing users to create, alter,
+and delete functions, as well as to reference them within queries.
+
+Presto by default supports a MySQL-based function namespace manager.
+It is pluggable and can be configured by an administrator.
+
+To instantiate a MySQL-based function namespace manager with id
+``prod``, add a file ``etc/function-namespace/prod.properties``
```

Comment:
Let's use maybe "test" or "example" rather than the word "prod". Maybe double check what other examples are using.